admin/services/data-access/sql.service.js

```diff
@@ -19,8 +19,11 @@ let cache = {}
  * @param {string} type
  * @return {string | undefined}
  */
-const findTediousDataType = (type) => Object.keys(TYPES).find(k => {
-  if (type.toUpperCase() === k.toUpperCase()) { return k }
+const findDataType = (type) => Object.keys(sqlService.TYPES).find(k => {
+  logger.debug(`findTediousDataType('${type}'): called`)
```

Comment:
I think this was meant to say `findDataType` instead of `findTediousDataType`?

Reply:
Yes, it should have.  Many thanks!